pymeasure/display/widgets/table_widget.py

```diff
@@ -0,0 +1,546 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+from numpy import float64, NaN
+from functools import partial
+import pyqtgraph as pg
+import pandas as pd
+
+from ..Qt import QtCore, QtWidgets, QtGui
+from .tab_widget import TabWidget
+from ...experiment import Procedure
+
+SORT_ROLE = QtCore.Qt.ItemDataRole.UserRole + 1
+SORTING_ENABLED = True
```

Comment:
Actually, I added it during debug to easily switch between sorting mode and non sorting mode.
I left it on purpose to allow future debug. 

I would avoid to ask users whether they want to enable sorting or not, since I can't anticipate any need to disable it.

Reply:
Ah that makes sense; I'm fine with that (but maybe adding a comment to note that it is left for debugging purpose would be nice). I see that you already removed it; either way fine with me.

Reply:
Ok, done